Packs/GoogleSafeBrowsing/Integrations/GoogleSafeBrowsingV2/GoogleSafeBrowsingV2_test.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+
```

Comment:
nice tests for the helpeprs.
please add a test for the main functionality - url_command. 
make sure to include: dbotscore context, reliability, handle of multiple urls.

Reply:
done